DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
Does it actually take substantially longer? IIRC `gulp build` is the rate-limiting step here. My general preference is to just remember one command that always results in a correct state, even if it's a little slower, but to each his/her own.